src/clj_sqs_extended/core.clj

```diff
@@ -72,43 +73,58 @@
   (.deleteQueue sqs-client url))
 
 (defn send-message
-  [sqs-client url message]
-  (.sendMessage sqs-client url message))
+  ([sqs-client url body]
```

Comment:
I can see why you renamed the `message` arg to `body`. since there's an overloading of the term `message` between here vs the java client. however, it would read better if for a fn `send-message` to have an argument named `message` so that it's obvious to the user that we're sending this var, which contains the data that's to be sent. so I suggest sticking with naming this argument `message`.